### ISSUE TITLE
Improperly nested unordered list

### ISSUE THREAD

**Issue by erinchambers** [CDRH/accessibility#20]
## Issue Description

<ul> and <ol> must only directly contain <li>, <script> or <template> elements

List is likely nested incorrectly (nested `<ul>` elements should be wrapped by a list item (`<li>...</li>`) instead of between two of them.

``` html
<ul>
  <li><strong>[environment]</strong> (development/production/other) Generally only production HTML is committed so we can track changes. The system allows for arbitrarily named environments, to cover use cases like providing different HTML files derived from the data formats</li>
  <ul>
    <li><strong>es</strong> – temporary location to store and troubleshoot Elasticsearch ingest files</li>
    <li><strong>html-generated</strong></li>
    <li><strong>solr</strong> – temporary location to store and troubleshoot Solr ingest files</li>
  </ul>
</ul>
```

### Impact/Severity

Serious

### Link to relevant page(s)

 - https://cdrhdev.unl.edu/log/2018/api/

### Screenshot or Video

N/A

---

## Remediation Steps (if known)

Replace above code with this:
``` html
<ul>
  <li><strong>[environment]</strong> (development/production/other) Generally only production HTML is committed so we can track changes. The system allows for arbitrarily named environments, to cover use cases like providing different HTML files derived from the data formats
    <ul>
      <li><strong>es</strong> – temporary location to store and troubleshoot Elasticsearch ingest files</li>
      <li><strong>html-generated</strong></li>
      <li><strong>solr</strong> – temporary location to store and troubleshoot Solr ingest files</li>
    </ul>
  </li>
</ul>
```

**Comment by erinchambers** [comment]
Fixed
